px-data-grid-number-renderer.html

```diff
@@ -1,17 +1,37 @@
 <link rel="import" href="../polymer/polymer-element.html">
 <link rel="import" href="px-data-grid-renderer-mixin.html">
+<link rel="import" href="px-data-grid-theme.html">
 
 <dom-module id="px-data-grid-number-renderer">
   <template>
-    <style>
+    <style include="px-data-grid-styles">
       :host {
         display: flex;
         align-items: center;
+        flex-grow: 1;
+      }
+
+      input[invalid] {
+        outline: 1px solid red;
```

Comment:
There is already class name for input in invalid state? Check their website.

Reply:
Done.